cl/search/management/commands/cl_index_parent_and_child_docs.py

```diff
@@ -124,20 +299,48 @@ def add_arguments(self, parser):
             help=f"The document type to index, only 'parent' or 'child' documents. "
             f"If not provided, parent and child documents will be indexed.",
         )
+        parser.add_argument(
+            "--update-from-event-tables",
+            type=str,
+            required=False,
+            choices=[member.value for member in EventTable],
+            help=f"The document type to update from event history tables. "
+            f"'docket' for dockets, 'de' for docket entries or 'rd' for "
+            f"RECAP Documents.",
```

Comment:
This text requires an update. Possible values are  "search.Docket" , "search.DocketEntry" , "search.RECAPDocument"

Reply:
Oh, yeah. This is now updated.